packages/aws-cdk-lib/aws-rds/lib/instance-engine.ts

```diff
@@ -315,6 +315,8 @@ export class MariaDbEngineVersion {
   public static readonly VER_10_3_37 = MariaDbEngineVersion.of('10.3.37', '10.3');
   /** Version "10.3.38". */
   public static readonly VER_10_3_38 = MariaDbEngineVersion.of('10.3.38', '10.3');
+  /** Version "10.3.39". */
+  public static readonly VER_10_3_39 = MariaDbEngineVersion.of('10.3.39', '10.3');
```

Comment:
October is not yet here 😄   https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MariaDB.Concepts.VersionMgmt.html#MariaDB.Concepts.VersionMgmt.EndOfLife103